web/src/pages/Upstream/locales/zh-CN.ts

```diff
@@ -15,80 +15,89 @@
+  'page.upstream.step.healthy.checks.active.http_path': '路径',
+  'page.upstream.step.input.healthy.checks.active.http_path': '请输入路径',
+  'page.upstream.step.healthy.checks.active.port': "端口",
+  'page.upstream.step.input.healthy.checks.active.port': '请输入端口',
+  'page.upstream.step.healthy.checks.active.host': '域名',
+  'page.upstream.step.input.healthy.checks.active.host': '请输入域名',
+  'page.upstream.step.healthy.checks.active.interval': '间隔',
+  'page.upstream.step.input.healthy.checks.active.interval': '请输入间隔',
+  'page.upstream.step.healthy.checks.successes': '成功次数',
+  'page.upstream.step.input.healthy.checks.successes': '请输入成功次数',
+  'page.upstream.step.healthy.checks.http_failures': '失败次数',
+  'page.upstream.step.healthy.checks.active.create.req_headers': '创建请求头',
+  'page.upstream.step.input.healthy.checks.http_failures': '请输入失败次数',
+  'page.upstream.step.healthy.checks.active.req_headers': '请求头',
+  'page.upstream.step.input.healthy.checks.active.req_headers': '请输入请求头',
+  'page.upstream.step.healthy.checks.passive': '被动健康检查',
+  'page.upstream.step.healthy.checks.passive.create.http_statuses': '创建状态码',
+  'page.upstream.step.healthy.checks.passive.http_statuses': '状态码',
+  'page.upstream.step.input.healthy.checks.passive.http_statuses': '请输入状态码',
+  'page.upstream.step.healthy.checks.passive.tcp_failures': 'tcp失败次数',
+  'page.upstream.step.input.healthy.checks.passive.tcp_failures': '请输入tcp失败次数',
+  'page.upstream.notificationMessage.enableHealthCheckFirst': '请先启用探活健康检查。',
 
-  'upstream.create.edit': '编辑',
-  'upstream.create.create': '创建',
-  'upstream.create.upstream.successfully': '上游成功',
-  'upstream.create.basic.info': '基础信息',
-  'upstream.create.preview': '预览',
+  'page.upstream.create.edit': '编辑',
+  'page.upstream.create.create': '创建',
+  'page.upstream.create.upstream.successfully': '上游成功',
+  'page.upstream.create.basic.info': '基础信息',
+  'page.upstream.create.preview': '预览',
 
-  'upstream.list.name': '名称',
-  'upstream.list.type': '类型',
-  'upstream.list.description': '描述',
-  'upstream.list.edit.time': '编辑时间',
-  'upstream.list.operation': '操作',
-  'upstream.list.edit': '编辑',
-  'upstream.list.confirm.delete': '确定删除该条记录吗？',
-  'upstream.list.confirm': '确定',
-  'upstream.list.cancel': '取消',
-  'upstream.list.delete.successfully': '删除记录成功',
-  'upstream.list.delete': '删除',
-  'upstream.list': '上游列表',
-  'upstream.list.input': '请输入',
-  'upstream.list.create': '创建',
+  'page.upstream.list.name': '名称',
+  'page.upstream.list.type': '类型',
+  'page.upstream.list.description': '描述',
+  'page.upstream.list.edit.time': '编辑时间',
+  'page.upstream.list.operation': '操作',
+  'page.upstream.list.edit': '编辑',
+  'page.upstream.list.confirm.delete': '确定删除该条记录吗？',
+  'page.upstream.list.confirm': '确定',
+  'page.upstream.list.cancel': '取消',
+  'page.upstream.list.delete.successfully': '删除记录成功',
+  'page.upstream.list.delete': '删除',
+  'page.upstream.list': '上游列表',
+  'page.upstream.list.input': '请输入',
+  'page.upstream.list.create': '创建',
```

Comment:
We have global keys  `component.global.delete` `component.global.operation` `component.global.name` `component.global.pleaseEnter` `component.global.edit` `component.global.cancel` `component.global.create` etc. we can consider simplifying some definitions ^^

Reply:
@guoqqqi Please file a new PR after this PR is merged.